acs-integration-tests/src/test/java/com/ge/predix/integration/test/PredixAuditIT.java

```diff
@@ -84,8 +87,12 @@ public void setup() {
     @Test(enabled = false)
     public void testAudit() throws Exception {
         long startTime = Instant.now().toEpochMilli();
-        this.zoneHelper.createTestZone(this.acsRestTemplateFactory.getACSTemplateWithPolicyScope(), "predix-audit-zone",
-                true);
+
+        String zoneId = "predix-audit-zone";
+        OAuth2RestTemplate acsAdminRestTemplate = this.acsitSetUpFactory.getAcsAdminRestTemplate(zoneId);
+        this.zoneFactory.createTestZone(acsAdminRestTemplate, zoneId,
+                Collections.singletonList(this.zoneTrustedIssuer));
+
         HttpHeaders headers = ACSTestUtil.httpHeaders();
         headers.setAccept(Collections.singletonList(MediaType.APPLICATION_JSON));
```

Comment:
@Test(enabled = false)
This test is not being executed since we don't have audit on integration environment. It would be wise  to put a todo  note to remind the re-enabling of  the test and test these changes

Reply:
Added TODO comment.